lib/mayaUsd/ufe/UsdUndoCreateGroupCommand.cpp

```diff
@@ -35,14 +35,14 @@ namespace ufe {
 
 UsdUndoCreateGroupCommand::UsdUndoCreateGroupCommand(
     const UsdSceneItem::Ptr& parentItem,
-#if (UFE_PREVIEW_VERSION_NUM < 3005)
+#ifndef UFE_V3_FEATURES_AVAILABLE
```

Comment:
This file is only included for Ufe v2. Here we don't want this extra selection arg for Ufe v2.